community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -971,22 +1110,27 @@ public PrimitiveIntIterator graphGetPropertyKeys( KernelStatement state )
     }
 
     @Override
-    public Property graphGetProperty( KernelStatement state, int propertyKeyId )
+    public boolean graphHasProperty( KernelStatement state, int propertyKeyId )
+    {
+        return graphGetProperty( state, propertyKeyId ) != null;
+    }
```

Comment:
Do we really need `graphHasProperty( KernelStatement state, int propertyKeyId )` if it doesn't provide a more efficient solution than `graphGetProperty( state, propertyKeyId )`? This will open up for something like:
```
if ( graphHasProperty( state, propertyKeyId ) )
{
    Object property = graphGetProperty( state, propertyKeyId );
}
 ```
Which will just be two calls for `graphGetProperty( state, propertyKeyId )`. Isn't it better to just force the usage to be the null check?